### ISSUE TITLE
Different nature of deltaE-E plots for same simulations in two different NPTOOL installed system.

### ISSUE THREAD

**Issue by mustak21** [adrien-matta/nptool#38]
Hi,
This is quite a strange problem that we are facing. We have NPTOOL installed in two of our pcs both having CentOS 7 and ROOT 6. When we simulate the reaction 7Be(d,p)8Be->alpha+alpha for same values of all parameters concerned and then plot the delta E-E curves(for annular detectors) in these two different pcs the plots observed are not identical but has some prominent differences! In one of the plots (named npt1)the proton band observed has many discontinuous small gaps alongwith the expected alpha band however the other plot (named npt2) the proton band is continuous alongwith the alpha band but there is another band above the alpha one which is unexpected! 
What could be the possible reasons for such discrepancy?
[npt1.pdf](https://github.com/adrien-matta/nptool/files/1063915/npt1.pdf)
[npt2.pdf](https://github.com/adrien-matta/nptool/files/1063917/npt2.pdf)

**Comment by adrien-matta** [comment]
Hi,

Thanks for your email. I did. I did not code the S3 my self, so I am not sure and will have to look Into the code. I think what happen is that in the second plot you got the outer part of the detector coded with a thicker first layer, so the same particle loose more energy in a thicker material. Do you have the commit number for both version? You get it with
git rev-parse HEAD

Cheers,
Adrien.

Nb: sorry to be slow answering in general, we are in the middle of mounting an experiment with little time available.

**Comment by kkabita** [comment]
Hi,

Since we have built S3 using the existing detector S1, I have now attached the plots that we received using S1 in place of S3. The differences still exist in here. The S1 and S1a are the same detectors with different thickness (S1-100 microns, S1a-1500 microns)

The commit number of both the versions along with the respective graphs obtained are :

cbde16e9d630dbcde1a34e7e01475c4221a7420d
[npt1.docx](https://github.com/adrien-matta/nptool/files/1068176/npt1.docx)

c1465673309ec2dc544f585cba83bd27c0f1c669
[npt2.docx](https://github.com/adrien-matta/nptool/files/1068179/npt2.docx)

I hope this will give a clear view of the problem.

**Comment by mustak21** [comment]
Hi,

Since we have built S3 using the existing detector S1, I have now attached the plots that we received using S1 in place of S3. The differences still exist in here. The S1 and S1a are the same detectors with different thickness (S1-100 microns, S1a-1500 microns)

The commit number of both the versions along with the respective graphs obtained are :

cbde16e9d630dbcde1a34e7e01475c4221a7420d
[npt1.docx](https://github.com/adrien-matta/nptool/files/1068204/npt1.docx)

c1465673309ec2dc544f585cba83bd27c0f1c669
[npt2.docx](https://github.com/adrien-matta/nptool/files/1068221/npt2.docx)

I hope this will provide a more clear view of the problem.
